### PR TITLE
Added Sidebar component to Layouts

### PR DESCRIPTION
**Pull request by goutham-subramanyam**
@edwinbbu _a Please review.

cc: @amaldinesh7 

Note: The avatar block will be updated once the Avatar component is added to v2.

### CODE REVIEW

**Comment by edwinbbu** [comment]
@goutham-subramanyam _a
1. Unable to expand once the sidebar is collapsed.
Screenshot 2021-07-27 at 10 59 15 AM
2. Do we need the background hover color in the selected navlink. It is not there in design.

Design in neetoAuth
Screenshot 2021-07-27 at 11 01 01 AM

File: lib/layouts/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import { Rating, LeftArrow } from "@bigbinary/neeto-icons";
+import classnames from "classnames";
+
+export const NeetoLogo = () => (
```
Comment (edwinbbu) [inline lib/layouts/Sidebar.js:6]: Can we move the logo to a separate file.

**Comment by edwinbbu** [comment]
@edwinbbu _a

**Comment by edwinbbu** [comment]
Removed the hover color on the active tab. Merging the PR.

**Comment by goutham-subramanyam** [comment]
@edwinbbu 

Screenshot 2021-07-27 at 3 32 31 PM

**Pull request merged**